.travis.yml

```diff
@@ -30,6 +30,8 @@ env:
     - DB=POSTGRES_10_4 PROJECT=CORE
     - DB=POSTGRES_10_4 PROJECT=WILDFLY
 
+install: mvn dependency:resolve
```

Comment:
Why not skip?

Reply:
Cleaner output from the build scripts. Dependencies are resolved and downloaded already when building the project.